docs/adrs/010-logs.md

```diff
@@ -0,0 +1,35 @@
+# ADR 010: What to do with logs?
+
+**Problem:** when commands fail, you usually want to look at the process logs to see what went wrong.
+You also might want to see logs as the system is building.
+This is more complicated than just "ok, so stream them" because we plan to add remote builders.
+How do we persist logs so you can look at them later, for example when a CI job fails?
+
+It seems like the best solution would be to have both a log store and a log sink.
+Specifically:
+
+1. during evaluation of jobs, logs can be streamed.
+   This would let us show them in the CLI or in a web UI.
+2. Logs could be stored like job artifacts in the content-addressable store.
```

Comment:
I think you want one more property. Some sort of toggle about what is display and at what verbosity.

Reply:
I think that'd probably be configured at the job level. We're just talking about stdout/stderr here. Maybe I should make that clearer up top!

Reply:
makes sense. Though likely you will want some way to dynamically pipe config to jobs. So an end user can request that job 42 be more verbose by only adding a command line flag to `rbt` rather than needing to update the job level config in the `.roc` file.